quota

```diff
@@ -49,9 +55,9 @@ print_lquota() {
     if [[ ${f[1]} -eq 0 && ${f[5]} -eq 0 ]];then
 	return # No usage by this user/group
     fi
-    spc=$(hum ${f[1]})
-    quot=$(hum ${f[2]})
-    lim=$(hum ${f[3]})
+    spc=$(hum ${f[1]} 1024)
+    quot=$(hum ${f[2]} 1024)
+    lim=$(hum ${f[3]} 1024)
```

Comment:
This seems a bit inconsistent, since we're using SI prefixes otherwise? Or wait, does this come from BLOCK_SIZE in mount.h being specified as 1024? If so, yes, this is ok..